Dangerfile

```diff
@@ -4,6 +4,7 @@ warn("Big PR") if git.lines_of_code > 1000
 xcov.report(
   workspace: 'RInAppMessaging.xcworkspace',
   scheme: 'RInAppMessaging-Example',
+  xccov_file_direct_path: 'artifacts/unit-tests/RInAppMessaging-Example.xcresult',
```

Comment:
(Idea) Maybe this can be passed as an argument in Danger command (I'm not sure if that's possible). That way we could use the merged .xcresult

Reply:
Maybe possible through env vars? I didn't include the combined xcresult though because the UI test's xcresult isn't set to generate a coverage report

Reply:
Ah right. Maybe I didn't turn the generation on because it was messing up total coverage. For now the current settings are ok